core/src/main/resources/hudson/model/View/index.jelly

```diff
@@ -44,11 +44,24 @@ THE SOFTWARE.
         <l:header>
             <!-- for screen resolution detection -->
             <l:yui module="cookie" />
-            <script>
-              YAHOO.util.Cookie.set("screenResolution", screen.width+"x"+screen.height, {
-                path: "/"
-              });
-            </script>
+            <j:invokeStatic var="req" className="org.kohsuke.stapler.Stapler" method="getCurrentRequest"/>
```

Comment:
I think you could use `request` directly. 

Source: https://github.com/stapler/stapler/blob/58a0fd5d7e336272a3c90140be632248b8f9a45c/jelly/src/main/java/org/kohsuke/stapler/jelly/DefaultScriptInvoker.java#L159

Reply:
I could.  I went looking for the documentation and did not see it exposed, obviously not looking hard enough :)